functionaltests/utils_test.go

```diff
@@ -90,15 +116,19 @@ func createCluster(t *testing.T, ctx context.Context, tf *terraform.Runner, targ
 	require.NoError(t, tf.Output("deployment_id", &deploymentID))
 	var apmID string
 	require.NoError(t, tf.Output("apm_id", &apmID))
-	var escfg esclient.Config
-	require.NoError(t, tf.Output("apm_url", &escfg.APMServerURL))
-	require.NoError(t, tf.Output("es_url", &escfg.ElasticsearchURL))
-	require.NoError(t, tf.Output("username", &escfg.Username))
-	require.NoError(t, tf.Output("password", &escfg.Password))
-	require.NoError(t, tf.Output("kb_url", &escfg.KibanaURL))
-
-	t.Logf("created deployment %s with APM (%s)", deploymentID, apmID)
-	return deploymentID, escfg
+	var output deploymentOutput
+	require.NoError(t, tf.Output("apm_url", &output.APMServerURL))
+	require.NoError(t, tf.Output("es_url", &output.ElasticsearchURL))
+	require.NoError(t, tf.Output("username", &output.Username))
+	require.NoError(t, tf.Output("password", &output.Password))
+	require.NoError(t, tf.Output("kb_url", &output.KibanaURL))
+
+	standaloneOrManaged := "standalone"
+	if enableIntegrations {
+		standaloneOrManaged = "managed"
+	}
+	t.Logf("created %s deployment %s with APM (%s)", standaloneOrManaged, deploymentID, apmID)
```

Comment:
This is not a "standalone deployment" but a "deployment with APM standalone"

```suggestion
	t.Logf("created deployment %s with %s APM (%s)", deploymentID, standaloneOrManaged, apmID)
```